mopidy-subsonic: init at 1.0.0

jgillich opened the pull request.

jgillich committed:
mopidy-subsonic: init at 1.0.0

jgillich commented:
```
ImportError: No module named 'connection'
```
Meh, works locally.

jgillich commented:
@domenkozar Would you mind taking a look? Thanks! :)

cillianderoiste commented on pkgs/applications/audio/mopidy-subsonic/default.nix:
```diff
@@ -0,0 +1,22 @@
+
+  meta = with stdenv.lib; {
+    homepage = http://www.mopidy.com/;
+    description = " Mopidy backend extension for playing music from Subsonic Music Streamer";
```
Nitpick: please remove the space at the start here.

jgillich commented:
@cillianderoiste Ops, fixed

cillianderoiste commented:
@jgillich thanks! Could it be that it's not compatible with python 3.4?

globin committed:
pythonPackages.py-sonic: fix SSL protocol usage

globin commented:
could you add mayflower@52c9a93 that fixes SSL handling in the subsonic library and test it again?

see https://github.com/crustymonkey/py-sonic/pull/13

jgillich commented:
I've added the patch, thanks! So again, this builds for me locally, don't know what's happening on Travis. I'm not actually (yet) using this because of https://github.com/rattboi/mopidy-subsonic/issues/15, but it at least it didn't show any errors when I added it to mopidy, so I thought I'd submit it anyway.
